.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: test
+
+on:
+  push:
+  pull_request:
+  # workflow_dispatch:
```

Comment:
Not sure if you like workflow_dispatch so I commented out. If you include it, you can click a button on the screen to run the Action whenever you want.